sdk/vc-data/src/data/courtRecordSearch/v1.ts

```diff
@@ -0,0 +1,92 @@
+import { VCV1, VCV1Subject } from '@affinidi/vc-common'
+
+import { PersonEV1, getBaseV1ContextEntries } from '../base'
+import {
+  CreateThing,
+  ExpandThing,
+  ExtendThing,
+  Type,
+  createContextEntry,
+  createVCContextEntry,
+  CreateExpandedThing,
+} from '../util'
+
+// Helper Types
+
+export type CourtRecordSearchQueryV1 = CreateThing<
+  'CourtRecordSearchQuery',
+  {
+    parent: PersonEV1
```

Comment:
Enforcing a `parent` seems a little too strict, but I'm also not sure what a "court record" is exactly so maybe it only applied to minors?

Reply:
this is to look up if a person has appeared in court or has any criminal history. I'll make parent optional here. it's just required by Idfy for their search